### DISCUSSION TITLE
@itwin/core-i18n localisation

### DISCUSSION THREAD

**Discussion by Rahul-0108** [iTwin/itwinjs-core#5705] [Q&A] [answered]
I have created 1 package and used  @itwin/core-i18n localisation and created a public folder while building the package and  placed the localisation file in it.
now I am installing the same package in another app, the localisation json data is not coming in network call instead the .html file of the container app is coming
![image](https://github.com/iTwin/itwinjs-core/assets/53996840/0400c195-4144-4fd9-b1d4-79e0bd18a48f)
I am using bentley/react-scripts . I think when using react-scripts start , the localisation json file should be copied from the external package app node_modules public folder into the the container app build folder when doing the development build. 

checking thr viewer-react project , looks like the same thing is done abd i have followed that
![image](https://github.com/iTwin/itwinjs-core/assets/53996840/8a0e946b-55dd-4119-9187-9d11844d183f)

can anyone pls help me what is going wrong and how can it be fix ?? and what goes on behind the scenes in bentley react scripts for localisation files handling in itwin viewer ??

**Comment by Rahul-0108** [comment]
Any thoughts on this ??

**Reply by pmconne** [reply]
@aruniverse and @calebmshafer probably know the answer off the top of their heads, but please be patient as there is a holiday in the US.

**Reply by Rahul-0108** [reply, marked as answer]
This is solved, Thanks

**Reply by aruniverse** [reply]
Can you explain what you did to solve the problem, so others who might run into this in the future can resolve it too?
